### PR TITLE
refactor: simplify data file handling

### PR DESCRIPTION
**Pull request by viceice**
ref: #5932

### CODE REVIEW

**Review by JamieMagee — APPROVED** [review]

File: lib/manager/pip_setup/extract.ts [hunk 1]
```diff
@@ -51,7 +51,8 @@ export async function extractSetupFile(
 ): Promise<PythonSetup> {
   const cwd = config.localDir;
   let cmd: string;
-  const args = [`"${join(__dirname, 'extract.py')}"`, `"${packageFile}"`];
+  const file = await resolveFile('data/extract.py');
```
Comment (rarkins) [inline lib/manager/pip_setup/extract.ts:54]: Are we still sure about `data/` ?

Comment (viceice) [reply lib/manager/pip_setup/extract.ts:54]: Are you suggesting an other folder? Currently I'm expecting files there

Comment (viceice) [reply lib/manager/pip_setup/extract.ts:54]: Or you you like to have them in manager folders?

Comment (rarkins) [reply lib/manager/pip_setup/extract.ts:54]: Seemed strange to have just one file in `data/` until now. And now I'm not sure that `extract.py` is a good fit for "data". Other than that I don't have an opinion.

Comment (viceice) [reply lib/manager/pip_setup/extract.ts:54]: We have two files there, the nodejs schedule json!

Comment (rarkins) [reply lib/manager/pip_setup/extract.ts:54]: ok, let's keep it as-is

**Pull request merged**